public/rooms/dormont/miratalk.html

```diff
@@ -2109,18 +2109,30 @@ <h2 id="b3">ACT 3 Intro</h2>
 
                 <h1 id="qna">Questions</h1>
 
-                    <ul>
-                        <li><b>ACT 1, 2, & 3: </b> 
-                            <span class="dialogue-name">Siffrin</span> 
-                            <span class="dialogue-expression">(questions1)</span>: (Questions...)
-                        </li>
-                        <li><b>ACT 4: </b> 
-                            <span class="dialogue-name">Siffrin</span>
-                            <span class="dialogue-expression">(fake1)</span>: (...)
-                        </li>
-                    </ul>
+                    <div class="dialogue">
```

Comment:
why'd you use notes and indents for just one siffrin portrait variation? i get adding the dialogue-head-s to the other ones but this is just One Single Thing different than the others

Reply:
good question you can change that if you want lol
i'm working on resolving merge conflicts rn btw ^^

Reply:
alrighty 👍 gonna do that then. thanks for looking at the merge conflicts yourself lmao. everything else looks good so far. iirc i didnt use relative links initially cuz i had problems with getting them to work or something? but it should by all means work so eh. dunno what happened back when